http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/monitor/LeoPubsubMessageSubscriber.scala

```diff
@@ -1541,6 +1541,7 @@ class LeoPubsubMessageSubscriber[F[_]](
           case e =>
             for {
               _ <- appQuery.updateStatus(msg.appId, AppStatus.Running).transaction
+              // You cannot update this log wording without updating the corresponding alert here https://console.cloud.google.com/monitoring/alerting/policies/8184448493858086363?project=broad-dsde-prod
```

Comment:
Thanks for giving the until testing a try, I agree this will suffice for now. 
One quit nit thought, could you update the comment to ask to update the log alert in staging as well?